test/reference1.deps.js

```diff
@@ -1,13 +1,14 @@
 module.exports = [ { block: 'b1' },
-  { modName: 'm1', modVal: true, block: 'b1', elem: 'e1' },
-  { modName: 'm1', modVal: 'v1', block: 'b1', elem: 'e1' },
-  { modName: 'bool', modVal: true, block: 'b1', elem: 'e1' },
+  { block: 'b1', elem: 'e1' },
```

Comment:
The only field in tests that changed (added) is this item:
`{ block: 'b1', elem: 'e1' }`
But looking at test case:
```
module.exports = {
    block: 'b1',
    content: [
        {
            elem: 'e1',
            elemMods: {
                m1: 'v1',
                bool: true
            },
            content: 'blah'
        },
...
```
I suppose it must be here. so it was a bug in this module.
Am I right?